keps/sig-node/4603-tune-crashloopbackoff/kep.yaml

```diff
@@ -0,0 +1,53 @@
+title: Tune Crashloop Backoff
+kep-number: 4603
+authors:
+  - "@lauralorenz"
+owning-sig: sig-node
+# participating-sigs:
+#   - sig-aaa
+#   - sig-bbb
+status: provisional
+creation-date: 2024-04-29
+reviewers:
+  - "@tallclair"
+  - "@pehunt"
+approvers:
+  - TBD
+  - "@oscar.doe"
```

Comment:
The approvers should be one of the sig-node's leads, see here: https://github.com/kubernetes/enhancements/blob/bd72947bfe7ef3531ae430bd730dcdd319bdec73/OWNERS_ALIASES#L75-L79

Reply:
Fixed in 08a81f2